setup.cfg

```diff
@@ -38,6 +38,7 @@ include_package_data = true
 install_requires =
     atpublic
     attrs
+    setuptools
```

Comment:
This should probably be a `tests_require` instead of `install_requires`.

Reply:
I'm not sure that's correct - we use setuptools on install... Why wouldn't that be a dep?

Reply:
`install_requires` is for it to be on the system at runtime. `setup_requires` is for installation itself, and you have a `tests_require` for dependencies for tests.

Reply:
Then it would be more appropriate it a setup_requires, no?

Reply:
Yep, you're right!